Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -161,15 +161,28 @@
 
     <!-- XTXT: risk card - High risk state - Days with high risk encounters -->
     <plurals name="risk_card_high_risk_encounter_days_body">
+        <item quantity="one">"Begegnungen an %1$d Tag mit erhöhtem Risiko"</item>
+        <item quantity="other">"Begegnungen an %1$d Tagen mit erhöhtem Risiko"</item>
+        <item quantity="zero">"Begegnungen an %1$d Tagen mit erhöhtem Risiko"</item>
```

Comment:
Do we need this? If there are no encounters with high risk, no red risk card should be displayed.

Reply:
This is true. 

But as written in https://github.com/corona-warn-app/cwa-app-android/pull/2236#discussion_r566878687 if we would get rid of the unnecessary quantities, `zero` would be gone anyway.

Reply:
Thx